docs/installation/direct-installation.md

```diff
@@ -19,42 +19,47 @@ git checkout main
 
 ```
 
-- Install dependencies: command `./scripts/direct-install/install-dependencies.sh`
+- Install dependencies command: `./scripts/direct-install/install-dependencies.sh`
 
-- Compile the project: command `./scripts/compile.sh`
+- Compile the project command: `./scripts/compile.sh`
 
-- Compile project: command `./scripts/direct-install/.sh`
+- Compile project command: `./scripts/direct-install/.sh`
```

Comment:
The command should be `./scripts/direct-install/install.sh` and does both Compiling and Installation. Check with devs.

Reply:
Asked David here: https://flarenetworks.slack.com/archives/C02NURDPAQZ/p1684866441918019